### DISCUSSION TITLE
Q&A

### DISCUSSION THREAD

**Discussion by zdsjjtTLG** [zdsjjtTLG/TrackIt#3] [Q&A]
Ask a question, start a conversation, or make an announcement

**Comment by Murphy666-code** [comment]
地图匹配之后没有生成匹配结果表
跑了五个小时 还能找到吗 还是得重新跑一遍
![屏幕截图 2025-04-10 171757](https://github.com/user-attachments/assets/770ff44b-740e-4a70-afe8-834629f3e3ea)

**Reply by zdsjjtTLG** [reply]
啊这，你这是最后存储，没有磁盘权限啊。那没办法，你应该先用一辆车的数据，匹配结束后，再存储测试一下。还有，以后跑这种大数据的，开INSTANT_OUTPUT=TRUE，每匹配完一辆车，他会自动存储

**Reply by Murphy666-code** [reply]
您好请问磁盘权限怎么调整 之前是可以存储的

**Reply by Murphy666-code** [reply]
查看了文件夹属性权限没问题啊 检查了一圈还是报错

**Reply by zdsjjtTLG** [reply]
试一下存当前目录./match_res.csv

**Reply by Murphy666-code** [reply]
我搞明白了 我的P1是文件夹的名称 应该写个csv文件名称

**Comment by ren-max11** [comment]
处理路网的时候   首先使用了环路处理  然后进行连通性修复  发现环路仍然被删除了    
![代码截图](https://github.com/user-attachments/assets/0783ff96-477a-400a-bf87-eca2b86985c2)
![路网截图](https://github.com/user-attachments/assets/cd31e0fd-abee-47d4-86eb-2325a409daec)

**Reply by zdsjjtTLG** [reply]
原始路网里面的环路的from_node和to_node是相同的吗，或者你把包含环路的部分路网发给我看看

**Reply by ren-max11** [reply]
原始只有线文件，第一步 ，  用create_node_from_link做出点文件（不做联通修复），产生的from_node和to_node不同
![image](https://github.com/user-attachments/assets/4afa4a52-df54-4213-843f-9ff7cc6b98fc)
第二步用环路处理  from_node和to_node也不一样
![image](https://github.com/user-attachments/assets/cd4664e3-5c8b-4239-a6dc-b4812ada12d2)
第三步  连通性修复   环路消失
![image](https://github.com/user-attachments/assets/03a00213-482b-4a7b-87e6-e5fce4f875e5)

**Reply by zdsjjtTLG** [reply]
![image](https://github.com/user-attachments/assets/66e0165a-5be1-4294-82aa-af25e642539d)

**Comment by zdsjjtTLG** [comment]
[circle_process.zip](https://github.com/user-attachments/files/20451571/circle_process.zip)


我处理这个示例，好像没有问题：

l = gpd.read_file(r'./LinkAfterModify.shp')
n = gpd.read_file(r'./NodeAfterModify.shp')
new_link, new_node = ng.NetReverse.circle_process(link_gdf=l, node_gdf=n)
new_link = new_link.to_crs('EPSG:4326')
new_node = new_node.to_crs('EPSG:4326')
new_link.to_file(r'F:\PyPrj\TrackIt\data\input\net\test\circle_process\aaa_link.shp')
new_node.to_file(r'F:\PyPrj\TrackIt\data\input\net\test\circle_process\aaa_node.shp')

**Comment by zdsjjtTLG** [comment]
![image](https://github.com/user-attachments/assets/13e6f873-435e-44b4-a55a-169c61b69e47)
